File: 0900-reordered-power-of-2/0900-reordered-power-of-2.cs
public class Solution {
    public bool ReorderedPowerOf2(int n) {
        string s=n.ToString();
        char[] charN = s.ToCharArray();
        Array.Sort(charN);
        s=new string(charN);
        for( ulong i=1;i<100000000000000;i=i*2){
            string x = i.ToString();
            char[] charX = x.ToCharArray();
            Array.Sort(charX);
            x = new string(charX);
            if(x==s){
                return true;
            }
        }
        return false;
    }
}

File: 0900-reordered-power-of-2/0900-reordered-power-of-2.cpp
class Solution {
public:
    bool reorderedPowerOf2(int n) {
        string s= to_string(n);
        sort(s.begin(),s.end());
        for(long long int i=1;i<=100000000000;i=i*2){
            string temp=to_string(i);
            sort(temp.begin(),temp.end());
            if(temp==s){
                return true;
            }
        }
        return false;
    }
};